scripts/error-extractor/ReactGlobPatterns.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
(By the way, this doesn't need to be called "React..."; only our actual runtime modules do because of providesModule. If you look at our other build scripts lots of them are just called "glob-patterns.js" or similar.)